### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kazumaki**

### CODE REVIEW

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @kazumaki and @EdSacasa  🌟

**Good work on this project, but we need to fix the issues mentioned bellow before moving on ⬇️** 

1. Please make the main video container fill up most of the screen width.

2. The elements in the aside section on the right should have thumbnails of the videos.

3. Set up Stickler to lint the CSS on your project, find out more on how to do that in [here](https://github.com/microverseinc/linters-config/tree/master/css)

4. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input)

5. Make the README file more descriptive, check [this](https://github.com/tundeiness/New-York-Times-clone/blob/master/README.md) link for reference

### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

**Comment by EdSacasa** [comment]
> ### Hello @kazumaki and @EdSacasa 🌟
> **Good work on this project, but we need to fix the issues mentioned bellow before moving on ⬇️**
> 
> 1. Please make the main video container fill up most of the screen width.
> 2. The elements in the aside section on the right should have thumbnails of the videos.
> 3. Set up Stickler to lint the CSS on your project, find out more on how to do that in [here](https://github.com/microverseinc/linters-config/tree/master/css)
> 4. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input)
> 5. Make the README file more descriptive, check [this](https://github.com/tundeiness/New-York-Times-clone/blob/master/README.md) link for reference
> 
> ### _After making the required changes please submit a new code review request_
> ### Good luck guys! 🍀

Hello Redvan,
We can see the thumbnail images for the suggested videos on our end, what type of change are you requesting or suggesting in the 2nd observation?
We took care of the other corrections.

Thank you.

File: index.html
Comment (wathigo) [inline index.html]: **[REQUIRED]** The suggested videos section images are not getting displayed because you have referenced the images wrongly. Change `images/thumb1.PNG` to `images/thumb1.png`

File: index.html
Comment (wathigo) [inline index.html]: **[REQUIRED]** The suggested videos section images are not getting displayed because you have referenced the images wrongly. Change `images/thumb1.PNG` to `images/thumb1.png`

File: index.html
Comment (wathigo) [inline index.html]: **[REQUIRED]** The suggested videos section images are not getting displayed because you have referenced the images wrongly. Change `images/thumb1.PNG` to `images/thumb1.png`

**Review by wathigo — CHANGES_REQUESTED** [review]
## HI Devs :wave: 
**Great effort so far,**
Before we get this approved, we have to fix a few things

1. Your CSS file has one error. Please upload your code to [w3 validator](https://validator.w3.org/#validate_by_uri) and fix it .
2. You have a lot of commits in the `master` branch and several merged Pull Requests. Don't merge a Pull Request that has not been approved. You should only have one Pull Request for this project. _Since you have done a lot of commits, make sure you don't do this in your next project._
3. Make sure you understand [githubflow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). In github flow, we don't have `develop` branch.
4. Give your Pull Request a title and a description which is a summary of the accomplished tasks.
5. You have some conflicts you need to resolve.
6. All files that are not important like `.DS_Store` and `.Rhistory` should be included in `.gitignore` file and not commited. 
7. Implement the changes pointed out below :point_down:


**[OPTIONAL]**
Make sure your code is well indented.

**After implementing the changes, Submit another request for review**
_Good luck :)_

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Review by Torres-ssf — APPROVED** [review]
### Hi there team :hand: 
Nice work implementing the requested changes :clap: 
### I'm approving this project :tada: 
Go on and merge to master.

Good luck with the next one. Happy coding :computer: 
### **IMPORTANT**
**[FOR NEXT PROJECTS]**

- For all HTML & CSS projects, use only **one** branch and do not merge it to master until it's been approved.

- As it said in the pathwrite, you should use Github Flow For all HTML and CSS projects. So give a feature name to your branch (e.g. embedding-images-and-video, html-forms, responsive-design, etc)  instead of develop/dev. You can read more about this [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/).

**Pull request merged**
